src/frontend/src/pages/DesignReviewView.tsx

```diff
@@ -0,0 +1,186 @@
+        availableUsers.set(time, usersAtTime);
+      });
+    });
+    return availableUsers;
+  }
+
+  function createUnavailableUsers() {
+    const allUsers = [...usersToAvailabilities.keys()];
+    for (let time = 0; time < numberOfTimeSlots; time++) {
+      const currentUsers = availableUsers.get(time) || [];
+      const currentUnavailableUsers = allUsers.filter((user) => !currentUsers.includes(user));
+      unavailableUsers.set(time, currentUnavailableUsers);
+    }
+    return unavailableUsers;
+  }
+
+  const handleOnMouseOver = (index: number) => {
+    setCurrentAvailableUsers(availableUsers.get(index) || []);
+    setCurrentUnavailableUsers(unavailableUsers.get(index) || []);
+  };
+
+  const handleOnMouseLeave = () => {
+    setCurrentAvailableUsers([]);
+    setCurrentUnavailableUsers([]);
+  };
+
+  const renderDayHeaders = () => {
+    return [
+      <TimeSlot backgroundColor="#D9D9D9" isModal={false} />,
+      daysOfWeek.map((day) => <TimeSlot key={day} backgroundColor="#D9D9D9" text={day} fontSize={'1em'} isModal={false} />)
+    ];
+  };
+
+  const renderSchedule = () => {
+    createAvailableUsers();
+    createUnavailableUsers();
+    return (
+      <Grid container>
+        {renderDayHeaders()}
+        {times.map((time, timeIndex) => (
+          <Grid container item xs={12} onMouseLeave={handleOnMouseLeave}>
+            <TimeSlot backgroundColor="#D9D9D9" text={time} fontSize={'1em'} isModal={false} />
+            {daysOfWeek.map((_day, dayIndex) => {
+              const index = dayIndex * times.length + timeIndex;
+              return (
+                <TimeSlot
+                  key={index}
+                  backgroundColor={getBackgroundColor(availableUsers.get(index)?.length, totalUsers)}
+                  onMouseOver={() => handleOnMouseOver(index)}
+                  isModal={false}
+                  icon={iconData.get(index)}
+                />
+              );
+            })}
+          </Grid>
+        ))}
+      </Grid>
+    );
+  };
+
+  const renderLegend = () => {
+    const colors = ['#D9D9D9', '#E0C0C1', '#E89A9B', '#E4797A', '#EF4345', '#D70C0F'];
+    return (
+      <Grid sx={{ display: 'flex', justifyContent: 'center' }}>
+        <Typography style={{ marginRight: '10px', fontFamily: 'oswald' }}>0/0</Typography>
+        {Array.from({ length: 6 }, (_, i) => (
+          <Box
+            sx={{
+              width: '1.5vw',
+              height: '1.5vw',
+              backgroundColor: colors[i]
+            }}
+          />
+        ))}
+        <Typography style={{ marginLeft: '10px', fontFamily: 'oswald' }}>
+          {totalUsers}/{totalUsers}
+        </Typography>
+      </Grid>
+    );
+  };
+
+  const renderAvailabilites = () => {
+    const fontSize = totalUsers > 10 ? '1em' : totalUsers > 15 ? '0.8em' : '1.2em';
+    return (
+      <Grid
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          backgroundColor: '#2C2C2C',
```

Comment:
If you call useTheme in here, and use this it should give you the same color: `theme.palette.background.paper`
We wanna avoid hard-coding colors (especially backgrounds) wherever we can